osu-winello.sh

```diff
@@ -114,6 +114,19 @@ Install()
 
       "$root_var" mkdir -p /etc/portage/sets
       printf " dev-vcs/git\n app-arch/zstd\n app-arch/p7zip\n net-misc/wget\n gnome-extra/zenity\n virtual/wine\n app-emulation/winetricks\n" | "$root_var" tee /etc/portage/sets/osu-winello
+      printf " media-libs/libsdl2 haptic\n" | "$root_var" tee /etc/portage/package.use/osu-winello
+      
+      if ! grep -q '@osu-winello' /var/lib/portage/world_sets; then
+      if ! emerge --info | grep -q 'ABI_X86="64 32"' && ! emerge --info | grep -q 'ABI_X86="32 64"'; then
+        echo
+        Info "Error: Unsupported configuration."
+        Info "Enable abi_x86_32 globally *or* manually emerge the @osu-winello set."
+        echo
+        Info 'To enable it globally, set ABI_X86="64 32" in /etc/portage/make.conf and run "emerge --newuse @world".'
+        Info "There *will* be circular dependencies to resolve!"
+        exit
```

Comment:
If you want to stop the script with that exit, instead of using `Info` and `Exit`, you could use the function `Error` which not only stops the script, but also reverts all the previous steps.

Something like: 
`Error: "There *will* be circular dependencies to resolve"`